### PR TITLE
feat(nat/transitip): add new sdk support

### PR DESCRIPTION
**Pull request by Lance52259**
**What this PR does / why we need it**:
New sdk supported: transit IP for private NAT

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format,
  will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
1. add new sdk support
```

### CODE REVIEW

File: openstack/nat/v3/transitips/results.go [hunk 1]
```diff
@@ -0,0 +1,42 @@
+package transitips
+
+import "github.com/chnsz/golangsdk/openstack/common/tags"
+
+type createResp struct {
+	// The transit IP detail.
+	TransitIp TransitIp `json:"transit_ip"`
```
Comment (ShiChangkuo) [inline openstack/nat/v3/transitips/results.go:7]: `request_id` is missing

File: openstack/nat/v3/transitips/results.go [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+	// The ID of the enterprise project to which the transit IP belongs.
+	EnterpriseProjectId string `json:"enterprise_project_id"`
+}
+
+type queryResp struct {
+	// The transit IP detail.
+	TransitIp TransitIp `json:"transit_ip"`
+}
+
+type updateResp struct {
```
Comment (ShiChangkuo) [inline openstack/nat/v3/transitips/results.go:39]: defined but not used

File: openstack/nat/v3/transitips/results.go (hunk 1 above)
Comment (Lance52259) [reply openstack/nat/v3/transitips/results.go]: done.

File: openstack/nat/v3/transitips/results.go (hunk 2 above)
Comment (Lance52259) [reply openstack/nat/v3/transitips/results.go]: this strucutre has been deleted.

**Pull request merged**